README.md

```diff
@@ -3,3 +3,28 @@ Common items for the different game converters. Including parsers, OS utilities,
 
 ## Current Status
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/54be2f027eb040448b8bf89410c7ff3d)](https://www.codacy.com/gh/ParadoxGameConverters/commonItems/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ParadoxGameConverters/commonItems&amp;utm_campaign=Badge_Grade)
+
+## Included Items
+
+### Color.h
+Represents a Paradox-defined color.
+
+Can be directly created in either the RGB or HSV color spaces.
+
+Can be imported in:
+* Unspecified with ints (becomes RGB)
```

Comment:
Add a simple example for all: color = { 1 2 3 }, color = hsv ( 0 0.4 0.2 } etc
Especially important at output so one knows what to expect..